Docs/MANUAL.md

```diff
@@ -0,0 +1,322 @@
+* **Coin Flips.** Flip coins and enter results.
+* **Die Rolls.** Roll six-sided dice and enter results.
+* **Card Draws.** Draw cards and enter results. (Be sure to replace cards and reshuffle as you draw, for the entropy calculation to be correct.)
+
+The easiest of these methods is certainly the "Quick Create", but in that case you are depending on the randomization of your mobile device, and if there is ever an exploit revealed, you'll be forced to sweep all of your funds. Using coin flips, die rolls, or playing cards is perhaps more likely to create good entropy, and is less likely to have an exploit revealed, but you _must_ properly flip every coin, roll every die, or draw every card, no matter how tedious it is, and you must wait until you have at least 128 bits of entropy, which is considered "Very Strong". If you are not willing to do this, you should just "Quick Create" instead. Once you have a "Very Strong" amount of entropy, you should click the "Done" button, and then you'll be given the opportunity to "Save" your new seed.
+
+The coin flipping, die rolling, and card drawing methods all have three buttons at the bottom, which allow you to: erase one entry; use the randomizer to add one entry; or use the randomizer to add all of the entries.
+
+The usage of coin, die, and card entropy in **Gordian Seed Tool** matches that of [**LetheKit**](https://github.com/BlockchainCommons/lethekit), so you can check the results for one on the other if you want to be sure of a new seed that you've created.
+
+<div align="center">
+  <table border=0>
+    <tr>
+      <td>
+        <a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-coin.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-coin.jpeg" width=250></a> 
+        <br><div align="center"><b>Coin Flips</b></div>
+      </center></td>
+      <td>
+        <a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-die.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-die.jpeg" width=250></a> 
+        <br><div align="center"><b>Die Rolls</b></div>
+      </center></td>
+      <td>     
+        <a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-cards.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-cards.jpeg" width=250></a> 
+        <br><div align="center"><b>Card Draws</b></div>
+      </center></td>
+    </tr>
+  </table>
+</div>
+
+
+## Storing a Seed
+
+Once you have a seed in **Gordian Seed Tool** it will be safely encrypted, and it will be securely backed up to iCloud as long as you've enabled iCloud's access to Keychain and iCloud Drive. 
```

Comment:
Do you say somewhere that it is possible to destroy all seeds including iCloud? If yes, there should be an internal link.

Reply:
You should point out that any other device logged into the same iCloud account and running Seed Tool will show the same seeds as the device on which they were created.

Reply:
Added details on iCloud (here) and on deleting everything (to the delete section, in addition to the settings section)